### ISSUE TITLE
Alt text for images on Instagram and Facebook does not pop up

### ISSUE THREAD

**Issue by deadscreem** [piroor/popupalt#9]
Facebook and Instagram have alt text for images that is supposed to describe what's in the image. The alt text is definitely present in the html, but it's not popping up.

**Comment by deadscreem** [comment]
Partially. On Instagram the alt text shows for the first image on the feed page, but once you scroll down to the 2nd (and beyond) the alt text no longer shows.

Facebook seems to work. But its' worth noting that the same bug as the above happens on facebook as well. however, that seems to be a facebook bug. because the alt text is missing in the html it seems for the 2nd and later images (it shows up when you click on the image however). However the alt text is present on insta for the 2nd and later images.

**Comment by deadscreem** [comment]
also, I just noticed, that on ig with viewing a users profile, it shows a list of image thumbnails. None of them show, but the alt is in the html.

**Comment by piroor** [comment]
I've done more changes at 4.2.0. I think the version should work on most cases.

**Comment by deadscreem** [comment]
Yes, everything seems to work now. If I notice any other cases I'll let you know.
Thanks for your good work!

**Comment by deadscreem** [comment]
I just noticed a bug related to this feature that happens with twitter and possibly else where. On twitter videos when you mouse over video player controls it causes a delay before the player button becomes responsive. Also on twitter when the page pops up a notice showing "See x new tweet(s)" clicking it causes slow loading. Disableing the option for this feature fixes it. Is this possible to be fixed? or is it a total limitation?

**Comment by piroor** [comment]
It is hard for me to find out testcases. Could you post bug report with certain steps to reproduce?

Anyway, changes at 8a138c6a1c7b17ead35e97df875201661ddb9537 (there is a typo in the commit message, I meant "simplify") may improve performance around this issue. After you add testcases, I'll try the changes is effective or not.

**Comment by deadscreem** [comment]
https://twitter.com/colbertlateshow has many videos on the page. videos auto play, but start muted. When you hover on the bottom of the video to make the controls pop-up and when the mute button is clicked there is sometimes a delay in unmuting, also happens with other buttons play/pause/restart.

**Comment by piroor** [comment]
Thank you for additional information. I couldn't experience any delay around the operation, so I think it has been fixed by recent commits.

**Comment by deadscreem** [comment]
Yes, I just checked. It is fixed in 4.2.1
